ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -128,9 +128,10 @@ class KtlintCommandLine {
 
     @Option(
         names = ["--color"],
-        description = ["Make output colorful"]
+        description = ["Make output colorful. To specify the color, use --color=\"RED\""],
+        arity = "0..1"
     )
-    var color: Boolean = false
+    var color: String? = null
```

Comment:
Changing type here will be a breaking change. 

I would propose to introduce additional parameter `--console-main-color` (short `-C`), that will be strongly typed `Color`: https://picocli.info/#_enum_types with default value `Color.NO_COLOR`. Additionally deprecate `--color` parameter.

Reply:
@Tapchicoma Do you think I may still run into the problem mentioned here where if I did `--console-main-color printAST` it would be interpreted as the param name? https://github.com/pinterest/ktlint/pull/585#issuecomment-534561994

I can test later to be sure, I'm still learning picocli.

Reply:
This particular problem comes from `arity = "0..1"` that allows to not have option value: https://picocli.info/#_arity. In the case above `printAST` is parsed as optional option value and not as subcommand.

My proposal for `--console-main-color` is to have mandatory option value (basically `arity = "1"`)

Reply:
Okay. Should be easy enough. I'm a little iffy about deprecating `--color`. The other alternative could be to introduce a new parameter, `--color-name` but then I think that's kind of annoying to ask the user to specify two parameters and do `--color --color-name RED`. 

Which would you prefer?

Reply:
Maybe yes, let's leave `--color` option that would just enable output coloring with default color (`RED`?) and `--color-name` that could be used to change color value.